src/wp-admin/plugins.php

```diff
@@ -414,7 +414,7 @@
 	'<p>' . __( 'The search for installed plugins will search for terms in their name, description, or author.' ) . ' <span id="live-search-desc" class="hide-if-no-js">' . __( 'The search results will be updated as you type.' ) . '</span></p>' .
 	'<p>' . sprintf(
 		/* translators: %s: ClassicPress Plugin Directory URL */
-		__( 'If you would like to see more plugins to choose from, click on the &#8220;Add New&#8221; button and you will be able to browse or search for additional plugins from the <a href="%s">ClassicPress Plugin Directory</a>. Plugins in the ClassicPress Plugin Directory are designed and developed by third parties, and are compatible with the license ClassicPress uses. Oh, and they&#8217;re free!' ),
+		__( 'If you would like to see more plugins to choose from, click on the &#8220;Add New&#8221; button and you will be able to browse or search for additional plugins from the <a href="%s">Plugin Directory</a>. Plugins in the Plugin Directory are designed and developed by third parties, and are compatible with the license ClassicPress uses. Oh, and they&#8217;re free!' ),
```

Comment:
Same here, if we go thru the hassle of stating they are license compatible we should probably mention that those are WP plugins and not recommended to be used on a CP install.

Reply:
Not wanting to get too deeply into this at this stage. I was just fixing it because it said "ClassicPress Plugin Directory" and that linked to the WP repo, so it was clearly wrong. Just tried to make it more generic. But it can be changed further if that's what others think.

Reply:
OK, I understand... so what about just making it clear that those are WP Plugins? 
`WordPress Plugin Directory` instead of `Plugin Directory`

Since it leads to WP Itself that is adequate, and at least it makes clear that this is not some sort of mixed, or CP plugin list/directory

I would go further and just add this:
`__( 'If you would like to see more plugins to choose from, click on the &#8220;Add New&#8221; button and you will be able to browse or search for additional plugins from the <a href="%s">WordPress Plugin Directory</a>. Plugins in the WordPress Plugin Directory are designed and developed by third parties, and are compatible with the license ClassicPress uses, but they are not necessarily designed for ClassicPress. Be sure to confirm their compatibility prior to install!' ),
`
Just a thought, tiny phrase that hopefully could avoid many future "issues"

Reply:
@KTS915 your thoughts?

Reply:
@bahiirwa This looks good to me!

Reply:
Look good to me too. I like that extra small phrase to clarify. Have done a commit to include this, plus reinstated the "WordPress Plugin Directory" usage.